packages/node/src/index.ts

```diff
@@ -65,6 +65,8 @@ const INTEGRATIONS = {
 
 export { INTEGRATIONS as Integrations, Handlers };
 
+export { trpcMiddleware } from './trpc';
```

Comment:
h: let's put this under `Handlers` for now. I don't think we need to worry about tree-shaking and this matches the pattern we have with the rest of the SDK. We can re-evaluate this at a later time.

Reply:
Done! f74e161b24ca9243bf0da0b1df1d06c47d664a4a